namecheap/namecheap_domain_record.go

```diff
@@ -90,7 +90,7 @@ func resourceNamecheapDomainRecords() *schema.Resource {
 						"ttl": {
 							Type:        schema.TypeInt,
 							Optional:    true,
-							Default:     1799,
+							Default:     1800,
```

Comment:
It does not. I mentioned my reasoning for it in the small novel of my description. 😅 

> I also tossed in a change you should feel free to ignore if it isn't valid or reasonable: the default ttl for records being 1799s seconds, or 29 minutes, 59 seconds. That just seemed strange, since even your web console offers standard values like 5, 10, 30 and 60 minutes. But you may have good reasons for using that value here. 🤷‍♂️